app/views/devise/sessions/new.html.erb

```diff
@@ -4,7 +4,7 @@
 
 <% if decorated_session.sp_name %>
   <div class='text-center'>
-    <%= image_tag(asset_url('user-access.svg'), width: '280', alt: '') %>
+    <%= image_tag(asset_url('user-access.svg'), width: '280', alt: '91') %>
```

Comment:
**question:** is this setting the alt text to "91"? @mitchellhenke did you mean to set the height?

Reply:
Good catch @orenyk ! I think this was a mistake from the changes discussed at https://github.com/18F/identity-idp/pull/7017#discussion_r982457649 .

Reply:
Oh wow, thanks for catching that 🤦🏼‍♂️ 

PR to fix here: https://github.com/18F/identity-idp/pull/7050